responsibleai/responsibleai/managers/causal_manager.py

```diff
@@ -269,6 +253,77 @@ def _whatif(self, id, X, X_feature_new, feature_name, y, alpha=0.1):
         return result._whatif(X, X_feature_new, feature_name,
                               y, alpha=alpha).to_dict(orient="records")
 
+    def request_global_cohort_effects(self, id, X_test):
+        """Get global causal effects for cohort data.
+
+        :param id: The query id for finding the
+                   causal config.
+        :type id: str
+        :param X_test: The data for which the global causal effects
+                       need to be generated.
+        :type X_test: Any
+        :return: An object of type CausalData with
+                 causal effects.
+        :rtype: CausalData
+
+        """
+        filtered = [r for r in self.get() if r.id == id]
+        if len(filtered) == 0:
+            raise ValueError(f"Failed to find causal result with ID: {id}")
+        result = filtered[0]
+        return result._global_cohort_effects(X_test)
+
+    def request_local_instance_effects(self, id, X_test):
+        """Get local causal effects for a given data point.
+
+        :param id: The query id for finding the
+                   causal config.
+        :type id: str
+        :param X_test: The data for which the local causal effects
+                       need to be generated for a given point.
+        :type X_test: Any
+        :return: An object of type CausalData with
+                 causal effects for a given point.
+        :rtype: CausalData
+        """
+        filtered = [r for r in self.get() if r.id == id]
+
+        if len(filtered) == 0:
+            raise ValueError(f"Failed to find causal result with ID: {id}")
+
+        if not isinstance(X_test, pd.DataFrame):
+            raise UserConfigValidationException(
+                'Data is of type {0} but it must be '
+                'a pandas DataFrame.'.format(type(X_test)))
+
+        if X_test.shape[0] > 1:
+            raise UserConfigValidationException(
+                'Only one row of data is allowed for '
+                'local causal effects.')
+
+        result = filtered[0]
+        return result._local_instance_effects(X_test)
+
+    def request_global_cohort_policy(self, id, X_test):
+        """Get global causal policy for cohort data.
+
+        :param id: The query id for finding the
+                   causal config.
+        :type id: str
+        :param X_test: The data for which the causal policy
+                       need to be generated.
```

Comment:
tiny nit:
```
need -> needs
```
(since causal policy is not plural)

Reply:
done